### PR TITLE
[wallet-ext] sign message modal updates

### PR DESCRIPTION
**Pull request by Nikhil-Mysten**
## Description 

based on https://www.figma.com/file/xu2VnyPAuruNi74UQzGigW/02-Wallet?type=design&node-id=8612-197309&mode=design&t=P5CLQQ15WK69MwI6-0

image
image

## Test Plan 

Locally
---
If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes section as highlighted during the release process.

### Type of Change (Check all that apply)

- [ ] protocol change
- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

**Comment by Nikhil-Mysten** [comment]
@pchrysochoidis I'm not quite sure how to test the qredo page but it seems like it will be impacted - if you have a a guide on how to do so that'd be great!

File: apps/wallet/src/ui/app/pages/site-connect/index.tsx [hunk 1]
```diff
@@ -88,16 +87,16 @@ function SiteConnectPage() {
 						rejectTitle="Reject"
 						onSubmit={handleHideWarning}
 						isWarning
-						addressHidden
+						address={activeAddress}
```
Comment (pchrysochoidis) [inline apps/wallet/src/ui/app/pages/site-connect/index.tsx:90]: I think showing the active address here as connected might be a bit confusing. At the time we show this warning the users doesn't have any connected accounts with the dapp and can chose later the accounts that will connect with

File: apps/wallet/src/ui/app/components/WalletListSelect.tsx [hunk 2]
```diff
@@ -113,6 +113,7 @@ export function WalletListSelect({
 				) : null
 			}
 			minimalPadding
+			boxShadow
```
Comment (pchrysochoidis) [inline apps/wallet/src/ui/app/components/WalletListSelect.tsx:116]: This component is used in other places as well and not sure if the shadow there is intended.
Screenshot 2023-06-26 at 12 06 34

**Review by pchrysochoidis — COMMENTED** [review]
Please have a look at the Transaction Approve flows it seems it affects the UI there as well

| before | after | 
| -- | -- |
| Screenshot 2023-06-26 at 12 20 34 | Screenshot 2023-06-26 at 12 18 38 |

Also about the Qredo http warning, should we update that one as well?

Screenshot 2023-06-26 at 11 56 38

and the Approve Connection view we probably need to add some shadow?

Screenshot 2023-06-26 at 11 56 26

You can use the `sdk/wallet-adapter/example` app to trigger the Qredo connection

File: apps/wallet/src/ui/app/pages/approval-request/SignMessageRequest.tsx [hunk 3]
```diff
@@ -47,13 +47,11 @@ export function SignMessageRequest({ request }: SignMessageRequestProps) {
 			scrollable
 		>
```
Comment (pchrysochoidis) [inline apps/wallet/src/ui/app/pages/approval-request/SignMessageRequest.tsx:48]: Do we need to add blended here as well?

Screenshot 2023-06-27 at 14 03 32

**Review by pchrysochoidis — APPROVED** [review]

**Pull request merged**